StructureDefinition/SfmJournalNoteExtension.StructureDefinition.xml

```diff
@@ -1,208 +1,220 @@
-<?xml version="1.0" encoding="utf-8"?>
 <StructureDefinition xmlns="http://hl7.org/fhir">
-  <id value="sfmjournalnoteextension" />
-  <url value="https://ehelse.no/fhir/StructureDefinition/SfmJournalNoteExtension" />
-  <version value="1.0.2" />
-  <name value="SfmJournalNoteExtension" />
-  <title value="SFM Journal note extension" />
-  <status value="active" />
-  <date value="2023-06-20" />
-  <description value="Extension for coded journal note canditades from SFM to EHR." />
-  <purpose value="Provide for necessary information on activities perfomed in SFM to the hosting EHR instance." />
-  <fhirVersion value="4.0.0" />
-  <kind value="complex-type" />
-  <abstract value="false" />
+  <id value="sfmjournalnoteextension"/>
+  <url value="https://ehelse.no/fhir/StructureDefinition/SfmJournalNoteExtension"/>
+  <version value="1.0.3"/>
+  <name value="SfmJournalNoteExtension"/>
+  <title value="SFM Journal note extension"/>
+  <status value="active"/>
+  <date value="2023-06-20"/>

```

Comment:
Shouldn't this be updated to the current date?

Reply:
Fixed!